### PR TITLE
Fix typo

### PR DESCRIPTION
**Pull request by benja-wu**

### CODE REVIEW

File: pkg/cluster/syncer.go [hunk 1]
```diff
@@ -102,7 +102,7 @@ func (s *Syncer) run(key string, prefix bool, send func(data map[string]*mvccpb.
 	defer watcher.Close()
 
 	// pull and send out full data copy. otherwise, if the application retrieves a copy
-	// of data at the beginnng, and if data is updated after the retrieval and before the
+	// of data at the beging, and if data is updated after the retrieval and before the
```
Comment (Loyalsoldier) [inline pkg/cluster/syncer.go:105]: beginning

File: pkg/filter/apiaggregator/apiaggregator.go [hunk 2]
```diff
@@ -246,7 +246,7 @@ func (aa *APIAggregator) handle(ctx context.HTTPContext) (result string) {
 func (aa *APIAggregator) newCtx(ctx context.HTTPContext, req *http.Request, buff *bytes.Buffer) (context.HTTPContext, error) {
 	// Construct a new context for the HTTPProxy
 	// responseWriter is an HTTP responseRecorder, no the original context's real
```
Comment (Loyalsoldier) [inline pkg/filter/apiaggregator/apiaggregator.go:248]: no -> not

File: pkg/util/texttemplate/texttemplate.go [hunk 3]
```diff
@@ -41,12 +41,12 @@ const (
 	WidecardTag = "{}"
 
 	// GJSONTag is the special hardcode tag for indicating GJSON syntax, must appear in the last
-	// of one template, if chose "{GJSON}", should provid antoher tag value at that level
+	// of one template, if chose "{GJSON}", should provider another tag value at that level
```
Comment (Loyalsoldier) [inline pkg/util/texttemplate/texttemplate.go:44]: provide

File: pkg/util/texttemplate/texttemplate.go [hunk 4]
```diff
@@ -270,29 +268,29 @@ func (t *TextTemplate) buildTemplateTree() error {
 			}
 		}
 	}
-	// every singl template is valid
+	// every singal template is valid
```
Comment (Loyalsoldier) [inline pkg/util/texttemplate/texttemplate.go:271]: single

File: pkg/filter/apiaggregator/apiaggregator.go [hunk 5]
```diff
@@ -245,8 +245,8 @@ func (aa *APIAggregator) handle(ctx context.HTTPContext) (result string) {
 
 func (aa *APIAggregator) newCtx(ctx context.HTTPContext, req *http.Request, buff *bytes.Buffer) (context.HTTPContext, error) {
 	// Construct a new context for the HTTPProxy
-	// responseWriter is an HTTP responseRecorder, no the original context's real
-	// repsonseWriter, or these Proxies will overwriten each others
+	// responseWriter is an HTTP responseRecorder, not the original context's real
+	// responseWriter, or these Proxies will overwrit each others
```
Comment (localvar) [inline pkg/filter/apiaggregator/apiaggregator.go:249]:
```suggestion
	// responseWriter, or these Proxies will overwrite each others
```

File: pkg/filter/proxy/proxy.go [hunk 6]
```diff
@@ -207,7 +207,7 @@ func (b *Proxy) reload() {
 	if len(b.spec.CandidatePools) > 0 {
 		var candidatePools []*pool
 		for k := range b.spec.CandidatePools {
-			candidatePools = append(candidatePools, newPool(b.spec.CandidatePools[k], fmt.Sprintf("backedn#candidate#%d", k),
+			candidatePools = append(candidatePools, newPool(b.spec.CandidatePools[k], fmt.Sprintf("backend#candidate#%d", k),
```
Comment (localvar) [inline pkg/filter/proxy/proxy.go:210]:
```suggestion
			candidatePools = append(candidatePools, newPool(b.spec.CandidatePools[k], fmt.Sprintf("proxy#candidate#%d", k),
```

File: pkg/util/circuitbreaker/circuitbreaker.go [hunk 7]
```diff
@@ -403,7 +403,7 @@ func (cb *CircuitBreaker) AcquirePermission() (bool, uint32) {
 	// always return true when closed.
 	// for time based window, failure rate or slow rate may change as time elapse,
 	// that's even no new result were recorded, state may transit from closed to
-	// open if may sucess results are evicted by time. but we just rely on the
+	// open if may success results are evicted by time. but we just rely on the
```
Comment (localvar) [inline pkg/util/circuitbreaker/circuitbreaker.go:406]:
```suggestion
	// open if success results are evicted by time. but we just rely on the
```

File: pkg/cluster/syncer.go (hunk 1 above)
Comment (benja-wu) [reply pkg/cluster/syncer.go]: Updated

**Review by localvar — APPROVED** [review]

File: pkg/object/meshcontroller/worker/api.go [hunk 8]
```diff
@@ -31,19 +31,19 @@ const (
 	meshNacosPrefix = "/nacos/v1"
 )
 
-func (wrk *Worker) runAPIServer() {
+func (worker *Worker) runAPIServer() {
```
Comment (xxx7xxxx) [inline pkg/object/meshcontroller/worker/api.go:34]: proposal `w`. [1]

[1] https://github.com/golang/go/wiki/CodeReviewComments#receiver-names

Comment (benja-wu) [reply pkg/object/meshcontroller/worker/api.go:34]: In some files, it's conflicted with `http.ResponseWriter `'s `w.`,e.g., 
```
func (worker *Worker) nacosRegister(w http.ResponseWriter, r *http.Request)
``` 
But I can replace the `woker` with `w`  in other not-conflicted files.

Comment (xxx7xxxx) [reply pkg/object/meshcontroller/worker/api.go:34]: I got it. then `worker` it is, for consistency.

**Review by xxx7xxxx — APPROVED** [review]

**Pull request merged**